tests/foreman/conftest.py

```diff
@@ -114,18 +115,16 @@ def log_test_execution(robottelo_logger, test_name):
     robottelo_logger.debug(f'Finished Test: {test_name}')
 
 
-@pytest.fixture
-def test_name(request):
-    """Returns current test full name, prefixed by module name and test class
-    name (if present).
-
-    Examples::
-
-        tests.foreman.ui.test_activationkey::test_positive_create
-        tests.foreman.api.test_errata::TestErrata::test_positive_list
+@pytest.fixture(autouse=True, scope="session")
+def relax_bfa():
+    """Relax BFA protection against failed login attempts
 
+    The robottelo.ssh.command should resolve the hostname for the current xdist worker
     """
-    return request.node._nodeid
+    ssh_command(
```

Comment:
Worth changing it back after the session?

Reply:
what is the risk if it isn't?

Reply:
For CI, none as the instances are immediately deleted.

For individuals running the tests ... not much, they just can't lock themselves out of their instance with failed logins anymore.  If they're trying to trigger that condition it won't.